src/main/drivers/pwm_output.c

```diff
@@ -174,17 +185,14 @@ void pwmEnableMotors(void)
 
 void pwmCompleteOneshotMotorUpdate(uint8_t motorCount)
 {
-    uint8_t index;
     TIM_TypeDef *lastTimerPtr = NULL;
 
-    for (index = 0; index < motorCount; index++) {
-
+    for (int index = 0; index < motorCount; index++) {
```

Comment:
Using an unsigned data type (e.g. `size_t`) should be a tad faster, at least in theory, and prefered over of `int`, because it avoids the conversion from signed to unsigned for `index < motorCount`.